website/docs/commands/export.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Export ⚡️
+sidebar_position: 27
+---
+
+Export outputs information about current project to a format used by external build tools such as SBT or Mill.
+Additionally the command supports JSON format for custom analysis of projects.
+
+The project configuration is read both from information specified in source files
+as well as options passed to the `export` command
+
+Let's take a simple one-file project as an example:
+```scala title=Hello.scala
+//> using scala "3.1.3"
+//> using option "-Xasync"
+//> using dep "com.lihaoyi::os-lib:0.9.0"
+
+object Hello {
+  def main(args: Array[String]): Unit =
+    println(os.pwd)
+}
+```
+
+# Exporting to SBT:
+
+```bash ignore
+scala-cli export Hello.scala --sbt
+```
+Note that `--sbt` is not required here since it's the default.
+
+The result is a sbt-compliant project created in the `dest/` directory:
+
+```
+dest
+├── project
+│   └── build.properties
+├── src
+│   └── main
+│       └── scala
+│           └── Hello.scala
+└── build.sbt
+```
+
+All the project's configuration resides now in `dest/build.sbt`:
+
+
+```scala title=dest/build.sbt
+scalaVersion := "3.1.3"
+
+scalacOptions ++= Seq("-Xasync")
+
+libraryDependencies += "com.lihaoyi" %% "os-lib" % "0.9.0" 
+
+libraryDependencies += "com.lihaoyi" %% "os-lib" % "0.9.0" % Test
+
+```
+
+
+To further configure the new SBT project you can pass options to the `export` command:
+- `setting` - extra SBT settings like XXXXX
+- `sbtVersion` - version of SBT used for the export, default is 1.6.1
```

Comment:
```suggestion
- `sbtVersion` - version of SBT used for the export, defaulting to the newest used by Scala CLI
```
`1.6.1` is waaaay old, we should keep this up-to-date... no point putting a particular version in the doc, either